tests/test_endpoint_common.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
No shebang needed here since it will be included in another script.. A shebang is the character sequence consisting of the characters number sign and exclamation mark (#!) at the beginning of a script.